### PR TITLE
Update Manage jobs doc, addresses docs for Feature#11439

### PR DESCRIPTION
**Pull request by currmie**
* Includes updates to User Manual > manage-jobs.rst file.
* See 'Clearing inactive jobs' section and the following note for these updates.

### CODE REVIEW

File: user-manual/administer/manage-jobs.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ Using the Jobs page
 
 When first arriving on the page, users will see a table showing the status of any
 ongoing jobs, as well as any previous jobs that have not been cleared by another
-user. See :ref:`below <jobs-permissions>` for more information on permissions and
+user. Jobs displayed include CLI tasks that you can run in a terminal. See :ref:`below <jobs-permissions>` for more information on permissions and
```
Comment (fiver-watson) [inline user-manual/administer/manage-jobs.rst:60]: 80-char line wrap here

File: user-manual/administer/manage-jobs.rst [hunk 2]
```diff
@@ -162,12 +162,14 @@ for startDate, endDate, jobName, jobStatus, jobInfo, and jobUser.
...
-completed or failed jobs) created by you from the Jobs page. Any jobs still
+completed or failed jobs) created by you from the Jobs page. This includes
+inactive jobs that you inititiated via command-line in a terminal. Any jobs still
 processing, or jobs created by another user, will not be affected.
 
 .. NOTE::
 
-   You can **only** clear jobs that you have initiated.
+   You can **only** clear jobs that you have initiated via the web-based
+   dashboard and via the command-line in a terminal.
```
Comment (fiver-watson) [inline user-manual/administer/manage-jobs.rst:172]: I think that only an admin can clear CLI-related jobs?

File: user-manual/administer/manage-jobs.rst [hunk 3]
```diff
@@ -219,4 +221,8 @@ users will only see jobs and be able to clear jobs which they have started. Only
 an :term:`administrator` can see the status of all jobs - however, like all users,
 administrators can only clear jobs that belong to them.
 
+.. NOTE::
+
+   As of 2.5, 
+
```
Comment (fiver-watson) [inline user-manual/administer/manage-jobs.rst:227]: This seems like an incomplete PR, perhaps? Note that this partial PR also ended up in your other PR!

**Review by fiver-watson — CHANGES_REQUESTED** [review]
This looks like it might have been incomplete? Trails off at the end. Note that the contents of this PR also ended up in your CLI task PR somehow - we'll have to sort that out!

File: user-manual/administer/manage-jobs.rst (hunk 3 above)
Comment (currmie) [reply user-manual/administer/manage-jobs.rst:227]: Weird. I must have decided to go a different direction with this and forgot to delete this. Good catch!

File: user-manual/administer/manage-jobs.rst (hunk 2 above)
Comment (currmie) [reply user-manual/administer/manage-jobs.rst:172]: Yes, only admins can clear CLI-related jobs. Will update, thx!

**Comment by currmie** [comment]
Edits made as per the review.

**Comment by currmie** [comment]
Closing this since it has been resolved with this commit 8e03a8b7e93d543cd9dfa5e0aeea5767f8581f27

**Pull request closed without merging**